src/resolve/resolved-dns-stub.c

```diff
@@ -977,7 +977,6 @@ static void dns_stub_process_query(Manager *m, DnsStubListenerExtra *l, DnsStrea
                 protocol_flags = SD_RESOLVED_DNS|SD_RESOLVED_NO_ZONE; /* Turn off mDNS/LLMNR for proxy stub. */
         } else if (DNS_PACKET_DO(p)) {
                 log_debug("Got request with DNSSEC enabled, enabling bypass logic.");
-                bypass = true;
```

Comment:
I cannot follow. Why? Please extend the commit message to explain why. Also, the log message is outdated. Please drop the whole `else if` branch.

Reply:
Oh, this was an old commit which i had reverted.